src/actions/getPlayers.js

```diff
@@ -0,0 +1,21 @@
+import { getPlayersPromise } from './helpers/getPlayersHelper'
```

Comment:
I think you no longer use this action and you use `getRoomInfo` instead of this , you use get .
If it's not correct, please let me know the flow.